rfcs/text/0024/ticket.yml

```diff
@@ -0,0 +1,151 @@
+  footnote: >
+    Ticket types include, but are not limited to:
+    Alarm, Event, Incident, Problem, Case, Bug, Feature Request, Task, Epic, User Story,
+    Knowledge Article, Change Request, etc.
+  type: group
+
+  fields:
+
+    - name: assignee
+      level: extended
+      type: keyword
+      short: Assigned to name
+      description: >
+        Name of the person or group with whom the ticket is currently assigned.
+      example: Sector_7-G_Safety_Desk
+
+    - name: created
+      level: core
+      type: date
+      short: Timestamp the ticket was created
+      description: >
+        Timestamp the ticket was created.
+      example: 2021-03-09T17:05:23.773Z
+
+    - name: description.long
+      level: core
+      type: text
+      short: Long description of the ticket
+      description: >
+        Long description / body of the ticket.  Typically contains all details and
+        specifics of the ticket.
+      example: >
+        A fiber cut has occurred in Springfield due to a down telephone pole.  All internet and
+        IP related services are down diddly offline. Customer call in-rates are elevated.
+
+    - name: description.short
+      level: core
+      type: text
+      short: Short description of the ticket
+      description: >
+        Short description / title of the ticket.  Typically, a brief overview.
+      example: >
+        Fiber cut in Springfield - internet service down
+
+    - name: details
+      level: extended
+      type: object
+      object_type: keyword
+      example: "{\"affected_version\": \"1.2.3\", \"env\": \"production\", \"resolution\": \"unresolved\", \"service\": \"Internet\"}"
+      short: Custom key/value pairs.
+      description: >
+        Custom key/value pairs.
+        Can be used to add meta information about the ticket, or include additional customized
+        ticket fields. Should not contain nested objects.
+        All values are stored as keyword.
+
+    - name: number
+      level: core
+      type: keyword
+      short: Ticket number
+      description: >
+        Numeric or alphanumeric unique identifier of the ticket; typically a ticket or record ID.
+      example: REQ-1234
+
+    - name: priority
+      level: extended
+      type: keyword
+      short: Priority of the ticket
+      description: >
+        Numeric or alphanumeric priority value assigned to the ticket.
+      example: P1
+
+    - name: requester
+      level: extended
+      type: keyword
+      short: Name of person or group whom the request is for
+      description: >
+        The name of the person or group that is making the request.  Can be different than the
+        person or group who submitted the ticket.
+      example: jdoe1234
+
+    - name: severity
+      level: extended
+      type: keyword
+      short: Severity of the ticket
+      description: >
+        Numeric or alphanumeric severity value assigned to the ticket.
+      example: 2 - High
+
+    - name: source
```

Comment:
`event.module` does make sense for this as the value is really intended to indicate the source ticketing system which houses said ticket.  

I hadn't really considered reusing fields from other blocks, especially, `event` since its description indicates it is for "log" or "metric" events which implies other fields from that block should be used as well in relation to the entry.